epub33/core/index.html

```diff
@@ -7850,178 +7850,179 @@ <h4>Content document dimensions</h4>
 				</section>
 			</section>
 
-			<section id="sec-reflowable-layouts">
-				<h3>Reflowable layouts</h3>
-
-				<p>Although control over the rendering of [=EPUB content documents=] to create <a
-						href="#sec-fixed-layouts">fixed layouts</a> is an obvious need not handled by other
-					technologies, there are also considerations for reflowable content that are unique to [=EPUB
-					publications=] (e.g., how to handle the flow of content in the [=viewport=]). This section defines
-					properties that allow [=EPUB creators=] to control presentation aspects of reflowable content.</p>
+		<section id="flow" data-epubcheck="true"
+			data-tests="https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L320,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L325,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L332,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L339">
 
-				<section id="flow" data-epubcheck="true"
-					data-tests="https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L320,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L325,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L332,https://w3c.github.io/epub-structural-tests/#08-layout_layout.feature_L339">
-					<h4>The <code>rendition:flow</code> property</h4>
+			<h3>Layout rendering control</h3>
 
-					<p>The <code>rendition:flow</code> property specifies the [=EPUB creator=] preference for how
-						[=reading systems=] should handle content overflow. </p>
+			<div class="proposed correction" id="change_2">
```

Comment:
This id should be "change_3". There are already two corrections annotated in the document, so the id conflict seems to be causing some really weird behaviour (I'm getting changes appearing in the annotation box).

Reply:
Thanks for catching this!

Reply:
Well, I didn't get it quite right... :(

There were actually three existing corrections. One issue I had to add two corrections for as it affected two different areas of the spec. It needs to be change_4